app/helpers/navbar_helper.rb

```diff
@@ -2,4 +2,12 @@ module NavbarHelper
   def active_class(link_path)
     current_page?(link_path) ? "active" : ""
   end
+
+  def settings_active_class(project)
```

Comment:
Why not do the inverse of this and add a class when it is what you expect? It's safer to have a white list than a black list in this case.